### PR TITLE
return error exit status in prometheus

### PR DESCRIPTION
**Pull request by mikeykhalil**
This should close this issue : https://github.com/prometheus/prometheus/issues/4293

### CODE REVIEW

File: cmd/prometheus/main_test.go [hunk 1]
```diff
@@ -155,3 +156,20 @@ func TestComputeExternalURL(t *testing.T) {
 		}
 	}
 }
+
+// Provide an invalid configuration file and verify the exit status indicates the error
```
Comment (brian-brazil) [inline cmd/prometheus/main_test.go]: Comments should be full sentences and end in a full stop.

**Comment by brian-brazil** [comment]
Thanks!

File: cmd/prometheus/main_test.go (hunk 1 above)
Comment (brian-brazil) [reply cmd/prometheus/main_test.go]: This is actually something that even varies across the Prometheus team, I wouldn't worry about it - especially for a tiny PR like this.

**Comment by mikeykhalil** [comment]
Thanks for the quick review!

**Pull request merged**